### DISCUSSION TITLE
Inquiries Regarding Revolt Userbase and API

### DISCUSSION THREAD

**Discussion by TejasLamba2006** [stoatchat/discussions#553] [Q&A] [answered]
My team is in the process of transitioning from Discord to Revolt, and I would appreciate your assistance in providing answers to some specific queries.

- Could you please provide information on the current userbase size of Revolt?
- How many servers or guilds are currently active on the Revolt platform?
- Can you share details regarding the number of bots or applications available on Revolt?

**Comment by nulldg** [comment, marked as answer]
globally, there are approximately 90,000 registered users (as of this post). check discover if you want to get a feel for the public servers and bots on the platform.

if you're creating a server or a bot, just do it. revolt isn't a get-rich-quick business opportunity. if your stuff is good, people will use it.

**Reply by TejasLamba2006** [reply]
I was thinking of creating a bot but I just wanted to know if its worth it or not!

Thanks for the numbers mate

**Reply by nulldg** [reply]
the process of creating bots on revolt is much smoother than on discord, and the revolt API is nicer to use than the discord API imo. when you're ready to publish your bot, join the revolt lounge and post the bot's invite link into `#Submit to Discover`. thank you and best of luck on your endeavor! :+1:
